graphql-social-media-service/service.bal

```diff
@@ -0,0 +1,118 @@
+import graphql_social_media.db;
+
+import ballerina/graphql;
+import ballerina/io;
+import ballerina/uuid;
+import xlibb/pubsub;
```

Comment:
@sameerajayasoma wdyt about using non ballerina/ballerinax package in standard sample?

Reply:
I will update the sample to use Kafka instead of pubsub.

Reply:
Removed pubsub and used Kafka.